benchmark/README.md

```diff
@@ -3,10 +3,111 @@
 
 # Benchmark
 
-This is a suite including several benchmarks with a set of models
-on Megatron and DeepSpeed.
+This is a suite including several benchmarks with a set of models on [Megatron-LM](https://github.com/NVIDIA/Megatron-LM) and [DeepSpeed](https://github.com/microsoft/DeepSpeed).
 
-## Single Node with Tensor Parallelism on Megatron
+## Preparation
+
+### Install Dependencies
+
+Please use the following command to install the dependencies. By default, [PyTorch](https://pytorch.org/) 2.0 is installed for better performance.
+
+```bash
+pip3 install torch torchvision transformers datasets matplotlib tabulate networkx triton pybind11
```

Comment:
nit: This might install the wrong pytorch in terms of CUDA version.

Reply:
Added clarification